src/Data/VarMap.hs

```diff
@@ -9,8 +9,7 @@ module Data.VarMap
   , (<>), mempty
   ) where
 
-import qualified Data.IntMap.Strict as Map
-import qualified Data.IntMap.Merge.Strict as Map
+import qualified Data.HashMap.Strict as Map
```

Comment:
Have we profiled this? I'm curious as to how the two perform.

Reply:
I haven't, but HashMaps are asymptotically more efficient than even PATRICIA trees. Since I was adding them anyway, I thought "why not change everything?".